### DISCUSSION TITLE
editing trainer sprites in pokemon clover

### DISCUSSION THREAD

**Discussion by genbread** [haven1433/HexManiacAdvance#183] [Q&A] [answered]
A lot of things seem to be broken when trying to edit pokemon clover, including the trainers. I am trying to change the sprites of the male protag. Any thoughts on how to fix it? Alternatives to this software are also welcome but I've tried like 6 others and most of them didn't work at all here.

**Comment by ShinyTillDawn** [comment, marked as answer]
From Clippy the bot:
> Hex Maniac Advance is not designed for editing other people's ROM hacks. Publishing someone else's ROM hack without permission is unfair towards the creator(s) of the original hack, even if you made significant changes to the hack. 

> Even if the ROM hack is being edited for personal use, you would have needed the developers' .toml file and knowledge on what things were changed and how the developers made such changes. Core game mechanics could have been modified with decomps or the CFRU (ex. hacks like Inclement Emerald and Radical Red), limiting the amount of tables that HMA can pick up and let you Goto. Nobody intended for such a hack to be modified nor examined any further. If you're having difficulties editing such a ROM hack, chances are, other people will struggle diagnosing your problems, let alone coming up with solutions.

> Nevertheless, you are free to look at the ⁠#faq for a few solutions to common problems that arise when editing an already modified Pokémon ROM.
